src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -279,6 +279,13 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
     return []; // No exports
   };
 
+  module.onRuntimeInitialized = () => {
+    if (!icuDataResource) {
+      // Use invariant culture if the app does not carry icu data.
```

Comment:
Maybe add a comment why this has to happen in `onRuntimeInitialized` and not in `preRun`?